public/assets/css/icons/email.css

```diff
@@ -0,0 +1,29 @@
+.icon .email {
+    width: 32px;
+    height: 33px;
```

Comment:
it's 32px, and you don't need to re put size. 
You already add this in icon.css

Reply:
remove width and height done